zodiac-cli/test/cli/linking/run

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -eux
+
+. $(dirname $0)/../core/setup.sh
+
+banner TSRP shared object dependencies
+#-------------------------------------
+
+nso="$(objdump -p $TSRP | grep -E '^\s+NEEDED' | wc -l)"
+[ "$nso" = "8" ] || exit 10
```

Comment:
Maybe. It could increase in the future, but I figure incrementing an integer in a test when we add a new solib dependency is a small price to pay for not occasionally breaking in prod after working in CI. More detailed reasoning [here](https://github.com/ambiata/ambiata-cli/pull/61#issuecomment-263776955), I did a similar thing for `ambiata-cli` (test is slightly pickier as we're actually distributing that executable).